targets/ath79-generic

```diff
@@ -449,6 +449,12 @@ device('ubiquiti-unifi-ap', 'ubnt_unifi', {
 	},
 })
 
+device('ubiquiti-nanobeam-m5-xw', 'ubnt_nanobeam-m5-xw', {
+	manifest_aliases = {
+		'ubiquiti-nanobeam-m5', -- upgrade from OpenWrt 17.01
```

Comment:
We leave these comments to indicate when to remove the aliases. As we don't support upgrades from any 17.01-based versions anymore, this comment would indicate that the alias should be removed now.

Reply:
How do you feel about leaving it in until the next release, in order to have a migration between the old and the new?

Reply:
If anyone still has a device based on Gluon 2018.1 or older, the update may subtly break the device and require a manual firstboot. It's better not to perform such updates automatically.

Reply:
Alright. Will redo the file tomorrow morning. Thanks for the feedback.

Reply:
resolved in 8690b98